omnigibson/prims/rigid_prim.py

```diff
@@ -52,12 +57,13 @@ class RigidPrim(XFormPrim):
 
     def __init__(
         self,
-        prim_path,
+        relative_prim_path,
         name,
         load_config=None,
     ):
         # Other values that will be filled in at runtime
         self._rigid_prim_view_direct = None
+        self._is_part_of_articulation = None
```

Comment:
super nit: maybe rename to slightly more formal language? e.g.: `belongs_to_articulation`?